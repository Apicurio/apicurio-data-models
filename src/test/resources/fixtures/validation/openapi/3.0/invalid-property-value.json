{
  "openapi": "3.0.2",
  "info": {
    "version": "1.0.0",
    "title": "Swagger Petstore",
    "description": "A sample API that uses a petstore as an example to demonstrate features in the OpenAPI 3.0 specification"
  },
  "servers": [
    {
      "url": "https://{username}.gigantic-server.com:{port}/{basePath}",
      "description": "The production API server",
      "variables": {
        "username": {
          "default": "demo",
          "description": "this value is assigned by the service provider, in this example `gigantic-server.com`"
        },
        "missingProperty": {
          "default": "foo",
          "description": "A missing property."
        },
        "port": {
          "enum": [
            "8443",
            "443"
          ],
          "default": "8443"
        },
        "basePath": {
          "default": "v2"
        }
      }
    }
  ],
  "paths": {
    "/enc-001": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "application/x-www-form-urlencoded": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8"
                },
                "profileImage": {
                  "contentType": "image/png, image/jpeg",
                  "headers": {
                    "X-Rate-Limit-Limit": {
                      "description": "The number of allowed requests in the current period",
                      "schema": {
                        "type": "integer"
                      }
                    }
                  }
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/enc-002": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "style": "form"
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/enc-003": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "explode": true
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/enc-004": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "allowReserved": true
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/enc-005": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "application/x-www-form-urlencoded": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "style": "matrix",
                  "explode": true,
                  "allowReserved": true
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/head-010": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "headers": {
                    "X-Header-1": {
                      "description": "The number of allowed requests in the current period",
                      "style": "form",
                      "schema": {
                        "type": "integer"
                      }
                    }
                  }
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/head-011": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "address": {
                    "type": "object",
                    "properties": {
                    }
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  },
                  "profileImage": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8",
                  "headers": {
                    "X-Header-1": {
                      "description": "The number of allowed requests in the current period",
                      "style": "simple",
                      "content": {
                        "text/plain": {
                          "schema": {
                            "type": "string"
                          }
                        },
                        "application/json": {
                          "schema": {
                            "type": "string"
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/link-002": {
      "get": {
        "responses": {
          "200": {
            "description": "the user being returned",
            "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "properties": {
                    "uuid": {
                      "type": "string",
                      "format": "uuid"
                    }
                  }
                }
              }
            },
            "links": {
              "address": {
                "operationId": "getUserAddress",
                "parameters": {
                  "userId": "$request.path.id"
                }
              }
            }
          }
        }
      }
    },
    "/mt-003": {
      "post": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  }
                }
              },
              "encoding": {
                "historyMetadata": {
                  "contentType": "application/xml; charset=utf-8"
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/op-009": {
      "get": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "requestBody": {
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string",
                    "format": "uuid"
                  },
                  "historyMetadata": {
                    "description": "metadata in XML format",
                    "type": "object",
                    "properties": {
                    }
                  }
                }
              }
            }
          }
        },
        "responses": {
          "200": {
            "description": "A successful response."
          }
        }
      }
    },
    "/op-013": {
      "get": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "responses": {}
      }
    },
    "/par-022": {
      "parameters": [
        {
          "name": "X-Header-Param",
          "in": "side"
        }
      ]
    },
    "/par-003/{id}": {
      "parameters": [
        {
          "name": "id",
          "in": "path"
        }
      ]
    },
    "/par-013/{id}": {
      "parameters": [
        {
          "name": "id",
          "in": "path",
          "required": true,
          "allowEmptyValue": false
        }
      ]
    },
    "/par-022": {
      "parameters": [
        {
          "name": "id",
          "in": "query",
          "allowEmptyValue": false,
          "style": "shallowObject"
        }
      ]
    },
    "/par-027/{id}": {
      "parameters": [
        {
          "name": "id",
          "in": "path",
          "required": true,
          "style": "form"
        }
      ]
    },
    "/par-023": {
      "parameters": [
        {
          "name": "criteria",
          "in": "query",
          "required": true,
          "style": "label"
        }
      ]
    },
    "/par-024": {
      "parameters": [
        {
          "name": "x-cookie",
          "in": "cookie",
          "required": false,
          "style": "label"
        }
      ]
    },
    "/par-025": {
      "parameters": [
        {
          "name": "X-Header",
          "in": "header",
          "required": false,
          "style": "label"
        }
      ]
    },
    "/par-028": {
      "parameters": [
        {
          "name": "X-Header",
          "in": "header",
          "required": false,
          "allowReserved": true
        }
      ]
    },
    "/par-029": {
      "parameters": [
        {
          "name": "object",
          "in": "query",
          "required": true,
          "style": "deepObject",
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string"
                  },
                  "name": {
                    "type": "string"
                  }
                }
              }
            },
            "text/plain": {
              "schema": {
                "type": "string"
              }
            }
          }
        }
      ]
    },
    "/sameNameDifferentInWithinPathItem": {
      "get": {
        "description": "Parameters with same name and different locations defined inline within operation - valid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "name": "status",
          "in": "header",
          "description": "Status param in header",
          "required": false,
          "schema": {
            "type": "string"
          }
        }
      ]
    },
    "/dupesWithinPathItemInlined": {
      "get": {
        "description": "Duplicate parameters (name & in combination) defined inline within path item - invalid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        }
      ]
    },
    "/dupesWithinPathItemInlineAndRefCombo": {
      "get": {
        "description": "Duplicate parameters (name & in combination) one defined inline within operation and other referenced - invalid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "$ref": "#/components/parameters/statusParam"
        }
      ]
    },
    "/dupesWithinPathItemIndirectReference": {
      "get": {
        "description": "Duplicate parameters (name & in combination) one defined inline within operation and other indirectly referenced - invalid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "$ref": "#/components/parameters/indirectRef"
        }
      ]
    },
    "/incorrectReferenceWithinPathItem": {
      "get": {
        "description": "Parameters (name & in combination) pointing to non-existent reference - invalid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "$ref": "#/components/parameters/missing"
        }
      ]
    },
    "/sameNameDifferentInWithinOperation": {
      "get": {
        "description": "Parameters with same name and different locations defined inline within operation - valid",
        "parameters": [
          {
            "name": "status",
            "in": "query",
            "description": "Status param in query",
            "required": false,
            "schema": {
              "type": "string"
            }
          },
          {
            "name": "status",
            "in": "header",
            "description": "Status param in header",
            "required": false,
            "schema": {
              "type": "string"
            }
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      }
    },
    "/dupesWithinOpInlined": {
      "get": {
        "description": "Duplicate parameters (name & in combination) defined inline within operation - invalid",
        "parameters": [
          {
            "name": "status",
            "in": "query",
            "description": "Status param in query",
            "required": false,
            "schema": {
              "type": "string"
            }
          },
          {
            "name": "status",
            "in": "query",
            "description": "Status param in query",
            "required": false,
            "schema": {
              "type": "string"
            }
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      }
    },
    "/dupesWithinOpInlineAndRefCombo": {
      "get": {
        "description": "Duplicate parameters (name & in combination) one defined inline within operation and other referenced - invalid",
        "parameters": [
          {
            "name": "status",
            "in": "query",
            "description": "Status param in query",
            "required": false,
            "schema": {
              "type": "string"
            }
          },
          {
            "$ref": "#/components/parameters/statusParam"
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      }
    },
    "/parameterOverridingPathItemDefinitionInlined": {
      "get": {
        "description": "Parameter defined inline at operation level overriding the one defined inline within pathItem - valid",
        "parameters": [
          {
            "name": "status",
            "in": "query",
            "description": "Status param in query (string)",
            "required": false,
            "schema": {
              "type": "string"
            }
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query (number)",
          "required": false,
          "schema": {
            "type": "number"
          }
        }
      ]
    },
    "/par-007/operation/{id}": {
      "get": {
        "description": "Parameter defined inline at operation level but not in path template - invalid",
        "parameters": [
          {
            "name": "id",
            "in": "path",
            "required": true
          },
          {
            "name": "missing",
            "in": "path",
            "required": true
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      }
    },
    "/par-007/pathItem/{id}": {
      "get": {
        "description": "Parameter defined inline at pathItem level but not in path template - invalid",
        "parameters": [
          {
            "name": "id",
            "in": "path",
            "required": true
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "missing",
          "in": "path",
          "required": true
        }
      ]
    },
    "/op-011/{id}/{sub}": {
      "get": {
        "description": "Path parameter defined in template but no matching parameter defined - invalid",
        "parameters": [
          {
            "name": "id",
            "in": "path",
            "required": true
          }
        ],
        "responses": {
          "default": {
            "description": "..."
          }
        }
      }
    },
    "/par-021": {
      "parameters": [
        {
          "name": "Content-Type",
          "in": "header",
          "required": true
        }
      ]
    },
    "/sch-002": {
      "get": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "responses": {
          "200": {
            "description": "A successful response.",
            "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "properties": {
                    "property-1": {
                      "$ref": "http://www.example.org/apis/foo.json#/components/schemas/S1"
                    }
                  },
                  "discriminator": {
                    "propertyName": "name"
                  }
                }
              }
            }
          }
        }
      }
    },
    "/sreq-002": {
      "get": {
        "description": "Creates a new pet in the store.  Duplicates are allowed",
        "responses": {
          "200": {
            "description": "A successful response."
          }
        },
        "security": [
          {
            "api_key": [
              "write:foo",
              "read:bar"
            ]
          }
        ]
      }
    }
  },
  "components": {
    "parameters": {
      "statusParam": {
        "name": "status",
        "in": "query",
        "description": "Status param in query",
        "required": false,
        "schema": {
          "type": "string"
        }
      },
      "indirectRef": {
        "$ref": "#/components/parameters/statusParam"
      }
    },
    "schemas": {
      "xml-002": {
        "type": "object",
        "properties": {
          "id": {
            "type": "integer",
            "format": "int32",
            "xml": {
              "attribute": true
            }
          },
          "name": {
            "type": "string",
            "xml": {
              "namespace": "http://example.com/schema/sample",
              "prefix": "sample",
              "wrapped": true
            }
          },
          "animals": {
            "type": "array",
            "items": {
              "type": "string"
            },
            "xml": {
              "namespace": "http://example.com/schema/animal",
              "prefix": "animal",
              "wrapped": true
            }
          }
        }
      }
    },
    "securitySchemes": {
      "api_key": {
        "type": "apiKey",
        "name": "api_key",
        "in": "header"
      },
      "ss-008": {
        "type": "invalid"
      },
      "ss-009": {
        "type": "apiKey",
        "in": "body",
        "name": "paramname"
      },
      "ss-017": {
        "type": "http",
        "scheme": "basic",
        "bearerFormat": "JWT"
      },
      "ss-016": {
        "type": "http",
        "scheme": "leveraged"
      }
    }
  }
}
