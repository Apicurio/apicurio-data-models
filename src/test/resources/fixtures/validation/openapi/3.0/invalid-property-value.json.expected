[ENC-001] |medium| {/paths[/enc-001]/post/requestBody/content[application/x-www-form-urlencoded]/encoding[profileImage]->headers} :: Headers are not allowed for "application/x-www-form-urlencoded" media types.
[ENC-002] |medium| {/paths[/enc-002]/post/requestBody/content[multipart/form-data]/encoding[historyMetadata]->style} :: Encoding Style is not allowed for "multipart/form-data" media types.
[ENC-003] |medium| {/paths[/enc-003]/post/requestBody/content[multipart/form-data]/encoding[historyMetadata]->explode} :: "Explode" is not allowed for "multipart/form-data" media types.
[ENC-004] |medium| {/paths[/enc-004]/post/requestBody/content[multipart/form-data]/encoding[historyMetadata]->allowReserved} :: "Allow Reserved" is not allowed for "multipart/form-data" media types.
[ENC-005] |medium| {/paths[/enc-005]/post/requestBody/content[application/x-www-form-urlencoded]/encoding[historyMetadata]->style} :: Encoding Style is an invalid value.
[HEAD-010] |medium| {/paths[/head-010]/post/requestBody/content[multipart/form-data]/encoding[historyMetadata]/headers[X-Header-1]->style} :: Header Style must be "simple".
[HEAD-011] |medium| {/paths[/head-011]/post/requestBody/content[multipart/form-data]/encoding[historyMetadata]/headers[X-Header-1]->content} :: Header content cannot have multiple media types.
[LINK-002] |medium| {/paths[/link-002]/get/responses[200]/links[address]->operationId} :: The Operation ID does not refer to an existing Operation.
[MT-003] |medium| {/paths[/mt-003]/post/requestBody/content[application/json]->encoding} :: Encoding is not allowed for "application/json" media types.
[OP-009] |medium| {/paths[/op-009]/get->requestBody} :: Request Body is not supported for GET operations.
[OP-013] |medium| {/paths[/op-013]/get->null} :: Operation must have at least one Response.
[PAR-022] |medium| {/paths[/par-022]/parameters[0]->style} :: Parameter Style must be one of: ["matrix", "label", "form", "simple", "spaceDelimited", "pipeDelimited", "deepObject"] (Found "shallowObject").
[PAR-023] |medium| {/paths[/par-022]/parameters[0]->style} :: Query Parameter Style must be one of: ["form", "spaceDelimited", "pipeDelimited", "deepObject"] (Found "shallowObject").
[PAR-003] |medium| {/paths[/par-003/{id}]/parameters[0]->required} :: Path Parameter "id" must be marked as required.
[PAR-013] |medium| {/paths[/par-013/{id}]/parameters[0]->allowEmptyValue} :: Allow Empty Value is not allowed (only for Query params).
[PAR-027] |medium| {/paths[/par-027/{id}]/parameters[0]->style} :: Path Parameter Style must be one of: ["matrix", "label", "simple"]  (Found "form").
[PAR-023] |medium| {/paths[/par-023]/parameters[0]->style} :: Query Parameter Style must be one of: ["form", "spaceDelimited", "pipeDelimited", "deepObject"] (Found "label").
[PAR-024] |medium| {/paths[/par-024]/parameters[0]->style} :: Cookie Parameter style must be "form". (Found "label")
[PAR-025] |medium| {/paths[/par-025]/parameters[0]->style} :: Header Parameter Style must be "simple". (Found "label").
[PAR-028] |medium| {/paths[/par-028]/parameters[0]->allowReserved} :: Allow Reserved is only allowed for Query Parameters.
[PAR-029] |medium| {/paths[/par-029]/parameters[0]->content} :: Parameter content cannot have multiple media types.
[PAR-019] |medium| {/paths[/dupesWithinPathItemInlined]/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemInlined]/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemInlineAndRefCombo]/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemInlineAndRefCombo]/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemIndirectReference]/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemIndirectReference]/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-018] |medium| {/paths[/incorrectReferenceWithinPathItem]/parameters[1]->$ref} :: Parameter Reference must refer to a valid Parameter Definition.
[PAR-019] |medium| {/paths[/dupesWithinOpInlined]/get/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinOpInlined]/get/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinOpInlineAndRefCombo]/get/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinOpInlineAndRefCombo]/get/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-007] |medium| {/paths[/par-007/operation/{id}]/get/parameters[1]->name} :: Path Parameter "missing" not found in path template.
[PAR-007] |medium| {/paths[/par-007/pathItem/{id}]/parameters[0]->name} :: Path Parameter "missing" not found in path template.
[OP-011] |medium| {/paths[/op-011/{id}/{sub}]/get->null} :: No definition found for path variable "sub" for path '/op-011/{id}/{sub}' and method 'GET'.
[PAR-021] |medium| {/paths[/par-021]/parameters[0]->name} :: The "Content-Type" header parameter will be ignored.
[SCH-002] |medium| {/paths[/sch-002]/get/responses[200]/content[application/json]/schema/discriminator->discriminator} :: Schema Discriminator is only allowed when using one of: ["oneOf", "anyOf", "allOf"]
[SREQ-002] |medium| {/paths[/sreq-002]/get/security[0]->null} :: Security Requirement 'api_key' scopes must be an empty array because the referenced Security Definition not "oauth2" or "openIdConnect".
[SVAR-003] |medium| {/servers[0]/variables[missingProperty]->null} :: Server Variable "missingProperty" is not found in the server url template.
[XML-002] |medium| {/components/schemas[xml-002]/properties[name]/xml->wrapped} :: XML Wrapped elements can only be used for "array" properties.
[SS-008] |medium| {/components/securitySchemes[ss-008]->type} :: Security Scheme Type must be one of: apiKey, http, oauth2, openIdConnect
[SS-009] |medium| {/components/securitySchemes[ss-009]->in} :: API Key Security Scheme must be located "in" one of: query, header, cookie
[SS-017] |medium| {/components/securitySchemes[ss-017]->bearerFormat} :: Security Scheme "Bearer Format" only allowed for HTTP Bearer auth scheme.
[SS-016] |medium| {/components/securitySchemes[ss-016]->scheme} :: HTTP Security Scheme must be one of: ["basic", "bearer", "digest", "hoba", "mutual", "negotiate", "oauth", "vapid", "scram-sha-1", "scram-sha-256"] (Found: 'leveraged')